### ISSUE TITLE
Upgrade dependencies for beta-4

### ISSUE THREAD

**Issue by ra0x3** [FuelLabs/fuel-indexer#1167]
- Blocked by https://github.com/FuelLabs/fuel-indexer/issues/1074
- fuel-core 0.20.x
- fuels-rs (latest version that's compatible with fuel-core 0.20)
- We should not merge this until the beta-4 network is released

**Comment by deekerno** [comment]
This is currently blocked by the latest versions of `fuel-tx` requiring `std` for serialization, which makes it incompatible with WASM. We leverage `fuel-tx` heavily in the construction of block explorer types as the types from `fuel-core` are also not WASM-compatible. 

More information can be found here: https://github.com/FuelLabs/fuel-vm/pull/533

**Comment by deekerno** [comment]
This has been unblocked and should be finished soon.

**Comment by deekerno** [comment]
This has been merged into `feature/beta-4-release`.
